app/services/auth_token_verifier.rb

```diff
@@ -3,9 +3,10 @@
 class AuthTokenVerifier
   attr_reader :errors
 
-  def initialize(token)
+  def initialize(token, member)
```

Comment:
I'm passing the member now because I preferred the check the way I implemented it - checking that the token exists and belongs to the member. I had to fetch the member anyway because I need to pass their AKID to the express donor status update, so I thought I might as well pass it to the verifier.